### DISCUSSION TITLE
Walter (ESP32-S3 + LTE-MN, NB-IoT and GNSS) in a small (PyCom GPy compatible) package

### DISCUSSION THREAD

**Discussion by daanpape** [micropython/micropython#12777] [Show and tell]
Hi,

I just found out about this forum and I wanted to show something that we have been working on in the last few months. In 2022 we had a design which as based on a PyCom GPy. Unfortunately these PyCom boards were phased out and many were left in the blank. That is why we have developed Walter, a footprint and software compatible upgrade for the PyCom GPy. We have also integrated lot's of improvements and we have writen our MicroPython LTE library as open source software. If you are interested, please subscribe on our Crowd Supply pre-launch page. Of course you can ask me any question!

https://www.crowdsupply.com/dptechnics/walter

[![walter-cta](https://github.com/micropython/micropython/assets/6665380/3cc76741-bb09-4c80-8998-765e03ff0b10)](![walter-cta](https://github.com/micropython/micropython/assets/6665380/3cc76741-bb09-4c80-8998-765e03ff0b10))

**Comment by mattytrentini** [comment]
It looks fantastic! Congrats and good luck on the crowd funding. 

FWIW I'll be sharing the Walter news at the [October Melbourne MicroPython Meetup](https://www.meetup.com/micropython-meetup/events/296616159/) tonight.

**Comment by daanpape** [comment]
Hi @mattytrentini thank you very much for your feedback. I will virtually attend the meeting through Zoom, so if you have any questions, just shoot :)

Kind regards,
Daan
